src/constants/chainInfo.ts

```diff
@@ -223,6 +224,31 @@ const CHAIN_INFO: ChainInfoMap = {
     color: '#35D07F',
     backgroundColor: '#34d07f1f',
   },
+  [SupportedChainId.AURORA]: {
+    networkType: NetworkType.L1,
+    blockWaitMsBeforeWarning: ms`10m`,
+    bridge: 'https://rainbowbridge.app/transfer',
+    docs: 'https://aurora.dev/',
+    explorer: 'https://explorer.aurora.dev/',
+    infoLink: 'https://info.uniswap.org/#/',
+    label: 'Aurora',
+    logoUrl: auroraLogo,
+    nativeCurrency: { name: 'Ether', symbol: 'ETH', decimals: 18 },
+    color: '#70d150',
+    backgroundColor: '#34d07f1f',
+  },
+  [SupportedChainId.AURORA_TESTNET]: {
```

Comment:
Does it show up in the UI? We currently do not support any tokens on Aurora Testnet.
If it does not show up then we can keep it.

Reply:
Not sure, since there were testnets i added this one too. I'll try on a local webapp